<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editorForm</class>
 <widget class="QWidget" name="editorForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1043</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>771</x>
     <y>520</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1021</width>
     <height>461</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QLineEdit" name="eventEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>751</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QComboBox" name="cbDay">
   <property name="geometry">
    <rect>
     <x>771</x>
     <y>480</y>
     <width>62</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QComboBox" name="cbMonth">
   <property name="geometry">
    <rect>
     <x>841</x>
     <y>480</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QComboBox" name="cbYear">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>480</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>520</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Загрузить из Excel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
